### ISSUE TITLE
high severity vulnerability in package serialize-javascript

### ISSUE THREAD

**Issue by PierreMano** [yahoo/serialize-javascript#93]
Visit https://go.npm.me/audit-guide for additional guidance

  High            Remote Code Execution

  Package         serialize-javascript

  Patched in      >=3.1.0

  Dependency of   react-scripts

  Path            react-scripts > terser-webpack-plugin > serialize-javascript

  More info       https://npmjs.com/advisories/1548

**Comment by TrySound** [comment]
Please rise the issue here https://github.com/facebook/create-react-app. React-scripts should upgrade terser plugin.

**Comment by okuryu** [comment]
@PierreMano This is the repository for serialize-javascript. A patched version of serialize-javascript is already available, so if you are using an outdated version, please upgrade to the latest version.

**Comment by PierreMano** [comment]
Thanks for the reply, i will try to update but i think i have already did it! If the problem persists i will raise the issue on the other link!

**Comment by PierreMano** [comment]
Rise**

**Comment by TrySound** [comment]
You can edit your messages. Do not need to spam.

**Comment by croraf** [comment]
The advisor wrongly says: 
```
{"foo": /1"/, "bar": "a\"@__R-<UID>-0__@"} was serialized as {"foo": /1"/, "bar": "a\/1"/}
```

This is not true since the version: 2.1.0 or so. The regex is serialized as `new RegExp("(1\")", "")`. 
So it will look like:
```
{"foo": new RegExp("(1\")", ""), "bar": "a\new RegExp("(1\")", "")}
```

Now, if the issue still happens with this taken into account I don't know.

**Comment by ImmortalTreearms** [comment]
Hello, I'm also receiving this vulnerability warning using Vue-WebPack. I've updated to serialize-javascript@4.0.0 with no luck in resolution. Apologies if this is the wrong place to raise this. I'm not sure where else I should go at this time.

Here is the message

  High            Remote Code Execution                                         

  Package         serialize-javascript                                          

  Patched in      >=3.1.0                                                       

  Dependency of   @vue/cli-service [dev]                                        

  Path            @vue/cli-service > copy-webpack-plugin >                      
                  serialize-javascript                                          

  More info       https://npmjs.com/advisories/1548

**Comment by croraf** [comment]
@ImmortalTreearms It has been fixed in this package. And the intermediate copy-webpack-plugin did made an update to the version of serialize-javascript without the bug. 

You need to open an issue in https://github.com/vuejs/vue-cli/issues to tell them that because of security issue they have to update their copy-webpack-plugin dependency.
